sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamSuite.scala

```diff
@@ -444,7 +444,9 @@ class StreamSuite extends StreamTest {
     val inputData = MemoryStream[String]
     val df = inputData.toDS().map(_ + "foo").groupBy("value").agg(count("*"))
     // Test StreamingQuery.display
+    val dir1 = Utils.createTempDir().getCanonicalFile
     val q = df.writeStream.queryName("memory_explain").outputMode("complete").format("memory")
```

Comment:
Why do we need this (and similar changes)? The original test is running fine with temp checkpoint.